### PR TITLE
Project ideas document

### PR DESCRIPTION
**Pull request by pwalsh**
Closes https://github.com/opentrials/opentrials/issues/400

### CODE REVIEW

**Comment by pwalsh** [comment]
@benmeg please review.

cc @vitorbaptista

File: docs/extras/project-ideas/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+### Basic outlier detection
+
+#### What
+
+There is much that can potentially be learned just by looking for outliers in a large dataset. Even without specific domain knowledge, building a set of queries to find outliers can be a first step in asking questions about the data. As a starting point, consider such possibilities around participation, location, age range or interventions for a particular condition.
+
+#### How
+
+Using the [OpenTrials query interface](https://query.opentrials.net/), build a set of queries designed for outlier detection across key data fields.
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md:94]: I don't think we should give people access to our production re:dash instance for now. At least not before we have a read-only replica of our database, or some other mechanism to make sure slow queries don't hit us too hard.

File: docs/extras/project-ideas/index.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+### Visualise publication lag
+
+#### What
+
+It is imperative that results of clinical trials are published. This is how knowledge is shared: about the effectiveness, or not, of a particular intervention for a particular condition, for example. Unfortunately, many trials have their results published with a long delay, or, simply do not have their results published at all.
+
+#### How
+
+Use the [OpenTrials API](http://api.opentrials.net/v1/docs/) to find trials with a large lag in publication of results, or, trials that have finished and have not yet published results. Build basic visualisations that show lag over time, or lag by condition, by treatment, by trial sponsor, by primary investigator and so on.
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md]: We don't have the date a trial's results were published in our DB, so this isn't possible yet.

File: docs/extras/project-ideas/index.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+#### Skills
+
+- Intermediate JavaScript and HTML
+- Basic knowledge of visualisations
+
+### Visualise the relationship between conditions and unpublished/published/ongoing trials
+
+#### What
+
+Certain conditions can be subject to clinical trials at different frequencies over time. Of course, there can be very good reasons for this, for example, a boost in clinical trials for Ebola around the Ebola crisis of 2014. Choose one or several conditions, and show information on a timeline for ongoing trials, trials that have finished and have published results, and trials that have finished yet do not have published results.
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md:123]: Similar to a previous comment, we just have the registration date, so it's a bit hard to do a timeline. We can, however, see the relationship between conditions and trial status, or how long a trial has been registered.

File: docs/extras/project-ideas/index.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+With whatever web design tools you are fluent in!
+
+#### Skills
+
+- Intermediate UI and UX
+- Basic HTML, CSS and Javascript
+
+### Some general solutions of use to domain experts
+
+- For all trials registered in two or more places, show the prespecified primary and secondary outcomes (side by side?), so that discrepancies can be found
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md:151]: We don't have the primary/secondary outcomes in a structured form yet.

File: docs/extras/project-ideas/index.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+- Search for all trials in a given geographical area, that are recruiting participants for a given condition, age, and gender (e.g. cancer trials in London for men aged 18-30).
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md]: We don't have the participants age range yet.

File: docs/extras/project-ideas/index.md [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+
+#### What
+
+In OpenTrials, we have [collectors](https://github.com/opentrials/collectors), which use modules and functions of Python code to get data from some external source into the OpenTrials warehouse database for further processing.
+
+There are many data sources available. So far we've build out the spine of the database with core clinical trial records, but there are tens of additional sources that can potentially be matched onto the clinical trials we know about, and provide value to the database.
+
+#### How
+
+See the [documentation for all existing collectors](https://github.com/opentrials/collectors/tree/master/docs/collectors) and start your own with a new data source! A good place to begin would be to see if any collectors have not yet been written for the 16 [WHO Primary Registries](http://www.who.int/ictrp/network/primary/en/).
```
Comment (vitorbaptista) [inline docs/extras/project-ideas/index.md]: There're 15 primary registries so far (not 16). Maybe we should use "scraping" somewhere, as it's clearer (to who knows what scraping is) what we're talking about.

File: docs/extras/project-ideas/index.md (hunk 1 above)
Comment (pwalsh) [reply docs/extras/project-ideas/index.md:94]: @vitorbaptista I'm leaving it here anyway. on this hackathon, you'll have a local re:dash anyway.

**Pull request merged**